CondTools/RPC/data/RPCTwinMuxLinkMapInput.txt

```diff
@@ -0,0 +1,300 @@
+TM YB+1_S1 0 LB_RB+1_S1_BP1A 1580	CH0: W+1/RB1out/1 Backward 	CH1: W+1/RB2out/1 Backward 	CH2: W+1/RB1in/1 Backward 
```

Comment:
I see, 
https://github.com/cms-sw/cmssw/pull/16106/files#diff-4dd27580d779189b1b8da4360af12566R10
thanks.
I am wondering if it shouldn't belong to the data repo instead. How frequently you plan to change it?